src/components/InfoPanel/InfoPanel.test.js

```diff
@@ -1,29 +1,37 @@
+import { render, screen } from "@testing-library/react";
 import configureStore from "redux-mock-store";
 import { Provider } from "react-redux";
-import { mount } from "enzyme";
```

Comment:
Was it difficult to make the switch?

Reply:
Honestly it was trivial. The only issue being that we have to have a mindset change from querying by class name to things like `role`, `text`, or the escape hatch of `testid`.